src/components/SearchField/SearchField.module.css

```diff
@@ -13,7 +13,7 @@
 /**
  * The Input component.
  */
-.search-field__input {
+.search-field > .search-field__input {
```

Comment:
Needs to be more specific than `.input` now

Reply:
this is one of those things we would not need if the search field component were a recipe. But as an actual component, I suppose the value should be using one of our eds-size tokens instead of just a fixed value